networkit/cpp/distance/SSSP.cpp

```diff
@@ -76,12 +74,23 @@ std::set<std::vector<node>> SSSP::getPaths(node t, bool forward) const {
 	return paths;
 }
 
+std::vector<node> SSSP::getNodesSortedByDistance(bool moveOut) {
+	if (!storeNodesSortedByDistance) {
+		throw std::runtime_error("Nodes sorted by distance have not been stored. Set storeNodesSortedByDistance in the constructor to true to enable this behaviour.");
+	} else if (nodesSortedByDistance.empty()) {
+		throw std::runtime_error("The container has already been moved or run() has not been called yet. Please call run() first.");
+	}
+	if (moveOut) {
+		auto movedNodes = std::move(nodesSortedByDistance);
+		nodesSortedByDistance.clear();
+		return movedNodes;
```

Comment:
Are you sure this works, in the sense that it is actually returning a rval reference to a non-empty vector?

Reply:
I double-checked and this should correctly return a rval ref.